applications/ShallowWaterApplication/python_scripts/postprocess/generate_gid_list_file.py

```diff
@@ -0,0 +1,64 @@
+from os import path, listdir, getcwd
+
+_list_extension = ".post.lst"
+_ascii_extension = ".post.res"
+_binary_extension = ".post.bin"
+
+def GenerateGiDListFile(first_dir = None, other_dirs = [], file_name = None):
```

Comment:
note that it is dangerous to use objects as default args
E.g. using this in CTors of Classes will result in the same object being used!
It is one of the nastier pitfalls of python

Reply:
thanks for the hint